frontend/public/components/utils/index.tsx

```diff
@@ -47,6 +47,7 @@ export * from './k8s-watcher';
 export * from './workload-pause';
 export * from './list-dropdown';
 export * from './status-icon';
+export * from './encode';
```

Comment:
I'd call the package `base64`

Reply:
I did originally, but thought that we might add different types of en/decoding and wanted to have them in single file... Although not sure what other will we add.